jobQueue.py

```diff
@@ -79,7 +79,7 @@ def add(self, job):
             self.log.info("add|JobQueue is full")
             return -1
         self.log.debug("add|Gotten next ID: " + str(job.id))
-        self.log.info("add|Unassigning job %s" % str(job.id))
+        self.log.info("add|Unassigning job ID: %d" % (job.id))
```

Comment:
True, I just wanted the job ID in the log message so that when we are viewing the logs, we can associate this specific message with a job ID. As it was previously, we would not be able to tell which job this message was associated with.